platform/viewer/cypress/support/commands.js

```diff
@@ -30,12 +30,73 @@ Cypress.Commands.add('openStudy', (patientName) => {
     cy.visit('/');
     cy.get('#patientName')
       .type(patientName);
-    cy.get('.studylistStudy > .patientName')
-      .contains(patientName)
-      .click();
+
+    const patientResult = '.studylistStudy > .patientName';
+
+    cy.get(patientResult)
+      .then({ timeout: 5000 }, ($patientResult) => {
```

Comment:
This would be a good place to leverage a Cypress `Alias` now that we have that tool in our toolbelt:

https://github.com/OHIF/Viewers/pull/968/files#diff-eee6caca0f3d24dfab61b837323f0192R83